### PR TITLE
Update Artifacts for CodeFlare Stack 0.0.4

### PR DESCRIPTION
**Pull request by anishasthana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,17 @@ It consists of three components:
 
 * [KubeRay](https://github.com/ray-project/kuberay) for management of remote Ray clusters on Kubernetes for running distributed compute workloads
 
-Intrgration of this stack into the Open Data Hub is owned by the Distributed Workloads Working
-Group. See [this page](https://github.com/opendatahub-io/opendatahub-community/tree/master/wg-distributed-workloads)
-for further details and how to get in touch.
+Intrgration of this stack into the Open Data Hub is owned by the Distributed Workloads Working Group. See [this page](https://github.com/opendatahub-io/opendatahub-community/tree/master/wg-distributed-workloads) for further details and how to get in touch.
```
Comment (KPostOffice) [inline README.md:17]:
```suggestion
Integration of this stack into the Open Data Hub is owned by the Distributed Workloads Working Group. See [this page](https://github.com/opendatahub-io/opendatahub-community/tree/master/wg-distributed-workloads) for further details and how to get in touch.
```

**Comment by jbusche** [comment]
Hi @anishasthana, I took a look at this... if I try applying your new subscription it works if I do the following:
1.   Take off any existing CodeFlare operator
2.  Apply your new subscription
```
oc apply -f https://raw.githubusercontent.com/anishasthana/distributed-workloads/codeflare_0.4.0/tests/resources/codeflare-subscription.yaml
```
3.  Approve the manual InstallPlan to be true
```
oc get ip
NAME            CSV                         APPROVAL   APPROVED
install-pkf49   codeflare-operator.v0.0.4   Manual     false
```
to
```
oc get ip
NAME            CSV                         APPROVAL   APPROVED
install-pkf49   codeflare-operator.v0.0.4   Manual     true
```
And then the codeflare operator starts up:
```
oc get pods 
NAME                                                       READY   STATUS    RESTARTS   AGE
codeflare-operator-manager-7b4fdcf579-5khjz                2/2     Running   0          80s
opendatahub-operator-controller-manager-84858b8998-2wctz   2/2     Running   0          7h9m
[root@api.jimlarge412.cp.fyre.ibm.com ~]# oc get csv
NAME                          DISPLAY                  VERSION   REPLACES                      PHASE
codeflare-operator.v0.0.4     CodeFlare Operator       0.0.4     codeflare-operator.v0.0.3     Succeeded
opendatahub-operator.v1.6.0   Open Data Hub Operator   1.6.0     opendatahub-operator.v1.5.0   Succeeded
```
I'm just not certain how the tests run anymore... looks like to run the `./run.sh distributed-workloads.sh` I need to have codeflare installed already, and I'm getting a weird 
```
./run.sh distributed-workloads.sh
tr: Illegal byte sequence
Creating namespace basictests-
error: resource(s) were provided, but no name was specified
error: A project named "basictests-" does not exist on "https://api.jimlarge412.cp.fyre.ibm.com:6443".
```
But then it runs until the notebook, but it's erroring out because it needs an API token... I think I've forgotten how to pass the API token to the notebook.

oc logs -f jupyter-nb-kube-3aadmin-0
```
You must obtain an API token by visiting https://oauth-openshift.apps.jimlarge412.cp.fyre.ibm.com/oauth/token/request
```

**Comment by jbusche** [comment]
Well, it's goofy for me, I've got multiple things going on...
1.  My cluster is too slow, so the mnist job is completing, but not in the 1200 seconds required.  I had to bump it up (put it to 2400 seconds for now)
2. The two curl commands to get the token aren't working, at least on my Mac.  I had to manually set the variable to my actual token in order to be able to run the command
3. The "*NotFound*" items aren't working for me, it appears to be "not found" on my cluster.  Not certain if this is due to my oc/kubectl version or not.
With all the above, it's taking a very long time to get through all the tests...  hopefully it'll run to completion now.

**Review by jbusche — APPROVED** [review]
LGTM

**Comment by jbusche** [comment]
Passed on my cluster with above tweaks - will try one last time from a Linux server just out of curiosity...

**Comment by jbusche** [comment]
Well, shoot!  That was easy - works great from a Red Hat 8 server.  I should have just done that instead of utilizing my Mac.
```
./run.sh took 1268 seconds
[INFO] Exiting with 0
```

**Review by MichaelClifford — APPROVED** [review]
LGTM!

**Pull request merged**